fs_overlay/etc/cont-init.d/00-welcome

```diff
@@ -2,6 +2,6 @@
 
 echo '
 ========================================
-HTTPS-PORTAL v1.23.1
+HTTPS-PORTAL v1.26.1
```

Comment:
HTTPS-PORTAL version does not really follow NGINX version. I will later change it to 1.24.